content/en/security/application_security/how-appsec-works.md

```diff
@@ -49,11 +49,9 @@ For Datadog ASM to be compatible with your Datadog configuration, you must have
 
 ### Serverless monitoring
 
-<div class="alert alert-info">Serverless threat monitoring is in private beta. Request to join the early preview with <a href="https://docs.google.com/forms/d/e/1FAIpQLScB3uSccf9lSAh7GcA8NZ8SsmUGQ5mi09DnDgZmqXcbiYfMzA/viewform">this form</a>.</div>
+<div class="alert alert-info">ASM support for AWS Lambda is in beta. Threat detection is done by using the Lambda extension.</div>
  
-Datadog ASM supports functions deployed on AWS Lambda. Detection is done by using the [Lambda extension][6]. 
-
-Full threat monitoring capabilities are available for Lambda functions. You can detect attackers targeting your functions, trace their attack path with deep code-level insights, and then remediate the threat.
+Datadog ASM for serverless provides deep visibility into attackers targeting your functions. With distributed tracing providing a context-rich picture of the attack, you can assess the impact and remediate the threat effectively. 
```

Comment:
Add a call to action link below pointing to the setup steps - https://docs-staging.datadoghq.com/kari/docs-4776-asm-serverless/security/application_security/getting_started/serverless/?tab=serverlessframework